src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,123 @@
+package duke.ui;
+
+import duke.Constants;
+import duke.TaskList;
+import duke.models.Task;
+
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Deals with interactions with the user and displaying output on the console
+ */
+public class Ui {
+    public Ui() {
+
+    }
+
+    /**
+     * Display welcome message to user
+     */
+    public void showWelcome() {
+        this.showIndentedDottedLines();
+        System.out.println(Constants.WELCOME_MESSAGE);
+        this.showIndentedDottedLines();
+    }
+
+    /**
+     * Display indented dotted line
+     */
+    public void showIndentedDottedLines() {
```

Comment:
I like the name of this method as it explains to the reader what the methods does at a sufficient level of detail.